ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -319,16 +315,11 @@ export const Homepage = withUserProfile()(class extends React.Component<
                   <div style={{display: 'flex', flexDirection: 'row', paddingTop: '2rem'}}>
                     <div style={styles.contentWrapperLeft}>
                       <div style={styles.mainHeader}>Researcher Workbench</div>
-                      <TooltipTrigger content={<div>Your Firecloud billing project is still being
```

Comment:
Do we have a way to tell when the Billing Buffer is exhausted, to display something similar?

Reply:
Not really. I think there's some design going into what to do there.

Reply:
Please add a story to capture that design and/or implementation activity if there's not one already.

Reply:
https://precisionmedicineinitiative.atlassian.net/secure/RapidBoard.jspa?rapidView=30&view=planning.nodetail&selectedIssue=RW-3424&epics=visible&selectedEpic=RW-1205